RecoTracker/LSTCore/src/alpaka/Triplet.h

```diff
@@ -381,7 +382,7 @@ namespace lst {
 
     //calculate residual
     if (layer3 <= 6 && ((side == lst::Center) or (drdz < 1))) { // for barrel
-      float paraA = r_init * r_init + 2 * (px * px + py * py) / (a * a) + 2 * (y_init * px - x_init * py) / a - r3 * r3;
+      float paraA = r_init * r_init + 2 * (px * px + py * py) / (a * a) + 2 * (y_init * px - x_init * py) / a - r_target * r_target;
```

Comment:
do I understand correctly that these are bugfixes for the (correct) selection of the hit positions depending on middle or outer MD used?
How does the performance change? My guess is that some of the cuts should be changed as well

Reply:
These are actually not bug fixes, since the target MD is always the 3rd MD in that block of code. I just wanted to make the variables consistent